prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,50 @@
+let number1;
+let number2;
+
+number1 = 10;
+number2 = 20;
+
+let product = number1 * number2;
+
+console.log(product);
+console.log(typeof product);
+
+let charge = 10;
+let payment = 50;
+let amountRemaining = payment - charge;
+
+console.log(amountRemaining);
+console.log(typeof amountRemaining);
+
+let tests = 90;
+let assignments = 85;
+let final = 95;
+let grade = (tests + assignments + final) / 3;
+
+console.log(grade);
+console.log(typeof grade);
+
+let firstName = 'Coby';
+let lastName = 'Welty';
+let fullName = firstName + ' ' + lastName;
+
+console.log(fullName);
+console.log(typeof fullName);
+
+let pH = 12;
+let isAcidic = pH < 7;
+
+console.log(isAcidic);
+console.log(typeof isAcidic);
+
+let numberOfSoldiers = 300;
+let isSparta = numberOfSoldiers === 300;
```

Comment:
All of these should be `const`. We only ever want to use `let` if we plan on reassigning the value at some point. since these variables values never change, we want to use `const` here instead. I always default to `const` and only ever change to a `let` when needed

Reply:
Got it, added a new commit replacing all `let` assignments with `const`